lib/presentation/views/notification/notification_info.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class NotificationInfoView extends StatelessWidget {
+  final String message;
+
+  const NotificationInfoView({Key? key, this.message = '\n \n \n blank'})
```

Comment:
Please do the same thing that I suggested for `AbousUsInfoView`

Reply:
done